haskell/yesod/README.md

```diff
@@ -0,0 +1,67 @@
+# yesod docker image
+
+Use this image as the base image for your yesod based applications.
+
+## Usage
+
+The simplest usage is to create a `Dockerfile` for your project that looks like
+this:
+```
+FROM thoughtbot/yesod:1.4.10-2
+
+RUN mkdir -p /app
+WORKDIR /app
+
+COPY *.cabal ./
+RUN cabal install --dependencies-only -j4
+```
+
+You can build your container like this:
```

Comment:
I think the point here was that this is how to set it up _without_ `docker-compose`. If you need database support, then you'd want to use `docker-compose`, and so should follow the next set of instructions. It's like the README version of a choose your own adventure. We should probably communicate this better though.